include/veriblock/blockchain/base_block_tree.hpp

```diff
@@ -340,6 +342,14 @@ struct BaseBlockTree {
     return validity_sig_.disconnect(id);
   }
 
+  index_t& getRoot() {
```

Comment:
In our terms this is called "bootstrap block" 
```suggestion
  index_t& getBootstrapBlock() {
```